web/embed-dart.html

```diff
@@ -55,10 +55,7 @@
   <body class="fullbleed vertical layout" unresolved>
     <div class="flex layout horizontal">
       <div class="relative vertical layout flex-6">
-        <div class="button-bar">
-          <paper-fab id="run-button" icon="av:play-arrow" mini></paper-fab>
-          <paper-fab id="export-pad" icon="launch" mini></paper-fab>
-        </div>
+        <dartpad-buttons></dartpad-buttons>
```

Comment:
I think we're better off having the buttons be separate components, that each embedder can position and style as they like.